core/lib/zksync_core_leftovers/src/temp_config_store/mod.rs

```diff
@@ -28,6 +32,12 @@ pub fn decode_yaml_repr<T: ProtoRepr>(yaml: &str) -> anyhow::Result<T::Type> {
     let this: T = zksync_protobuf::serde::deserialize_proto_with_options(d, false)?;
     this.read()
 }
+
+pub fn read_yaml_repr<T: ProtoRepr>(path_buf: PathBuf) -> anyhow::Result<T::Type> {
+    let yaml = std::fs::read_to_string(path_buf).context("failed decoding  YAML config")?;
+    decode_yaml_repr::<T>(&yaml)
+}
+
 //
 // TODO (QIT-22): This structure is going to be removed when components will be responsible for their own configs.
```

Comment:
Off-topic (?): Is this comment still relevant? With the god config approach, I don't think we'll ever get to this point. Or will this TODO be naturally resolved when `core_leftovers` are removed?